### PR TITLE
improve documentation

### LINKED ISSUE #662: Add Go to README dependencies
**What did you do?**

Follow the "Initial setup" instructions found in the repo's README .
Run the command `mkdir -p $GOPATH/src/github.com/18F`

**What did you expect to see?**

The terminal successfully create a new directory

**What did you see instead?**

`GOPATH` is not set

*Include a high-level description of the feature or error here*

Since we are relying on Go in our installation instructions and not everyone may have it installed and/or have the paths exported correctly, we should list along with the other `Dependencies` in the "Initial setup" portion of the README.

**Additional Information**

 - What browser and version are you using?
 - What operating system?
Mac OS

### PR DESCRIPTION
**Pull request by afeld**
Fixes #662.

Various cleanup - see individual commits. Easier to review ignoring whitespace.

### CODE REVIEW

File: docs/SPRINTS.md [hunk 1]
```diff
@@ -1,268 +1,342 @@
+_Deprecated._
```
Comment (afeld) [inline docs/SPRINTS.md]: @ryanhofdotgov @michaelccata Should we just delete this file?

File: README.md [hunk 2]
```diff
@@ -67,8 +52,6 @@ For more information on licenses and third-party source code please refer to the
 Clone the repository and `cd` into it:
 
 ```shell
-mkdir -p $GOPATH/src/github.com/18F
-cd $GOPATH/src/github.com/18F
```
Comment (afeld) [inline README.md]: #662

File: README.md [hunk 3]
```diff
@@ -205,18 +192,18 @@ All of the documentation may then be found in the respective directories under `
 
 #### Formatting
 
-JavaScript files are formatted using [Prettier](https://prettier.io/), though note this should only be done when a file is new or heavily modified. You should install Prettier for whatever editor you use.
+Supported files are formatted using [Prettier](https://prettier.io/), though note this should only be done when a file is new or heavily modified. You should install Prettier for whatever editor you use.
```
Comment (afeld) [inline README.md]: Also use it for Markdown, SASS, etc.

**Comment by hursey013** [comment]
Small thing (and happy to help clean up), some of the section in the table of contents no longer exist or are not linked properly:

- Getting to know the code
- Running the application
  - Setup
  - Reset locked app submission
- Docker containers

**Comment by afeld** [comment]
Ah, true. I'm tempted to take the Table of Contents out completely, as they get out-of-date easily. Thoughts?

**Comment by afeld** [comment]
Ditto for `docs/DEPENDENCIES.md`.

**Comment by hursey013** [comment]
@afeld I'd vote for taking them out just because they are a pain from a maintenance perspective, or maybe alternatively just make they way less granular and just hit the high points like `Development` and `Contributing`.

**Comment by afeld** [comment]
I know there were Reasons ™️ for having the DEPENDENCIES list, so waiting to hear from @ryanhofdotgov on that one.

**Comment by ryanhofdotgov** [comment]
Documenting "all dependencies (and licenses for dependencies)" was a contractual requirement in the QASP https://github.com/18F/bpa-opm-eqip/blob/master/QASP.md. So before the vendor went off contract, they put that list together. 

I think we have tools like Snyk (or CodeClimate, at the time this doc was prepared) that track the dependencies, but they didn't summarize the licenses.

TrueTandem wrote a tool to generate this list. We could run it periodically in the future instead and remove it from the repo for now. It was more important when we thought the project might not receive active development.

File: docs/SPRINTS.md (hunk 1 above)
Comment (ryanhofdotgov) [reply docs/SPRINTS.md]: +1 to delete. We have it elsewhere in Google Drive (though not as concisely).

**Pull request merged**